client/src/components/layout/ReviewTile.js

```diff
@@ -0,0 +1,12 @@
+import React from "react"
+
+const ReviewTile = ({ id, rating, comment, createdAt, updatedAt }) => {
+    return (
+        <li>
+            Rating: {rating}/5<br/>
```

Comment:
ideally we avoid `br` break tags and instead use CSS to help with formatting - this keeps all styling/formatting centralized. If you want these 2 pieces of info to be listed separately, considering using another HTML element like a `p` tag for the comment

Reply:
removed break and added p tags